src/com/facebook/buck/android/AndroidBundleDescription.java

```diff
@@ -243,12 +244,13 @@ public void findDepsForTargetFromConstructorArgs(
       ImmutableCollection.Builder<BuildTarget> targetGraphOnlyDepsBuilder) {
     javacFactory.addParseTimeDeps(targetGraphOnlyDepsBuilder, null);
     javaOptions.get().addParseTimeDeps(targetGraphOnlyDepsBuilder);
+
+    Optionals.addIfPresent(proGuardConfig.getProguardTarget(), extraDepsBuilder);
```

Comment:
So this is the only change in this PR that is actually fixing the logic, right? Is the rest of the changes for consistency with AndroidBinary?

Reply:
@styurin right